image_proc/src/edge_aware.cpp

```diff
@@ -1,7 +1,7 @@
 /*********************************************************************
 * Software License Agreement (BSD License)
 * 
-*  Copyright (c) 2008, Willow Garage, Inc.
+*  Copyright (c) 2019, Andreas Klintberg.
```

Comment:
I don't have an issue the other places you added a copyright to yourself, but this is a little too far here - you added a 2 returns lines without any functional change. Please remove it here.

Reply:
Of course, I'll add it back everywhere as suggested above.

Reply:
Awesome, thanks for understanding. Just want to make sure everyone's above board. :robot:

Reply:
I think that the most appropriate thing to do would be to append to copyright, rather than replace in many of these cases.

Reply:
I know that's more correct but personally I find that really cluttering. Since you both bring it up, lets do it.  

But good to keep us all honest, I'll default to the majority.

Reply:
I'm all fine with keeping it as it is Willow Garage being the copyright holder, really not an issue to me at all. Let me know if you decide on just Willow and I'll change it back. Right now its all consistent at least.

Reply:
I think the proper form is:

    Copyright (c) 2008, 2019, Willow Garage, Inc., Andreas Klintberg

Though I'm still looking for the source of where I saw this.

Reply:
Can you please add WG back to this one in one format or another? I couldn't find the source but what you did in other files with just the current year and WG/yourself is fine.